lib/remote/httpserverconnection.cpp

```diff
@@ -239,8 +240,6 @@ bool HandleAccessControl(
 		auto headerAllowOrigin (listener->GetAccessControlAllowOrigin());
 
 		if (headerAllowOrigin) {
-			CpuBoundWork allowOriginHeader (yc);
```

Comment:
This object has side-effects, so you have to be careful when using it. It's used to label the work that's done until it's destroyed (or `Done()` is called).